events/2022-google-summer-of-code/index.markdown

```diff
@@ -0,0 +1,60 @@
+---
+author: Mark Moll
+comments: false
+date: 2022-02-17 00:00:00 -0700
+layout: page
+slug: 2022-google-summer-of-code
+title: 2022 Google Summer of Code
+---
+# 2022 Google Summer of Code
+
+MoveIt is planning to participate again in the [Google Summer of Code](https://summerofcode.withgoogle.com). Below is a tentative list of project ideas. If you have your own project idea, feel free to create a pull request on the [repository for this page](https://github.com/ros-planning/moveit.ros.org). Unlike previous years, there are now two different project sizes. Medium projects are 175 hours, while large projects are 350 hours. See the [Google Summer of Code FAQ](https://developers.google.com/open-source/gsoc/faq) for details.
+
+## Project ideas
+
+### Bullet collision checking integration
+
+* Bullet support is partially done, but more work is needed. Support for continuous collision checking would be nice.
```

Comment:
There is actually support for continuous collision checking, it just is somewhat incomplete. Like all of the rest of the bullet plugin. https://github.com/ros-planning/moveit/pull/2897

Other open PRs include https://github.com/ros-planning/moveit/pull/2897

Adding more unit tests would be another important aspect and more on the functional side I think there are huge hacks wrt multithreading

Reply:
I'll add these links. @simonschmeisser would you be willing to be a mentor for this?

Reply:
I'm all in!